nox-utils/src/tmlt/nox_utils/_release.py

```diff
@@ -0,0 +1,167 @@
+"""Tools for creating releases."""
+
+import datetime
+import re
+from pathlib import Path
+
+from git import Repo
+from git.refs.remote import RemoteReference
+from git.refs.tag import Tag
+from git.remote import Remote
+from nox import Session
+
+
+def build_push_release(
```

Comment:
nit: This name sounds like "build a release wheel, and push it to pypi". Maybe `create_release_commits` or similar?

Reply:
Sure, I think that's just as good.